src/vs/workbench/api/common/extHostTerminalService.ts

```diff
@@ -51,8 +51,7 @@ export interface IExtHostTerminalService extends ExtHostTerminalServiceShape, ID
 export interface ITerminalInternalOptions {
 	isFeatureTerminal?: boolean;
 	useShellEnvironment?: boolean;
-	isSplitTerminal?: boolean;
-	target?: TerminalLocation;
+	resolvedExtHostTerminal?: ExtHostTerminal;
 }
```

Comment:
Is it possible to move location to the actual options instead of internal options?

Reply:
not possible because the types of location differ between MainThread and extHostTerminalService, but they share the TerminalLaunchConfig interface